.github/pull_request_template.md

```diff
@@ -14,8 +14,8 @@ Describe how you tested your change. If you are fixing a bug, please provide the
 
 - [ ] You've updated the CHANGELOG following the steps [here](https://github.com/klaviyo/magento2-klaviyo#making-updates)
 - [ ] **Internal Only** - If this is a release, have you:
-  - [ ] Updated the links in the CHANGELOG to point towards the new versions
-  - [ ] Incremented the version in the following places: module.xml and composer.json
+  - [ ] The links in the changelog have been updated to point towards the new versions
+  - [ ] The version has been incremented in the following places: module.xml and composer.json
```

Comment:
Line 16 wording needs to be updated to adjust for new wording of lines 17 and 18

Reply:
May be outside the scope of this PR, but w/ the use of changelogger should all the wording here be adjusted in general as we won't be manually updating these files anymore?

Reply:
Definitely agree with that -this should be updated. I'll do that as a part of this PR